### PR TITLE
WIP: doc update

### PR DESCRIPTION
**Pull request by rabernat**
I am starting to update the documentation page. My goal is to phase out the jekyll blog at pangeo-data.github.io and replace it with the sphinx-based site.

### CODE REVIEW

**Comment by mrocklin** [comment]
Generally this looks great to me.  

Thoughts on the background color when hovering over the TOC?  Currently it's a rather bright pink which is somewhat jarring next to the smooth green-blue gradient.

**Comment by mrocklin** [comment]
No sidebar, just a navbar on the top of the screen.  

You might want to try `make clean` and also maybe `git status` to make sure that you don't have anything lingering.

**Comment by rabernat** [comment]
Nothing lingering, clean checkout of branch:

image

**Comment by mrocklin** [comment]
I definitely don't see the sidebar.  I like it though.  The way the navbar is handling nested things isn't as intuitive to me.  One thought, maybe you're using different versions of `sphinx_bootstrap_theme` ?

**Comment by tjcrone** [comment]
Where is this site hosted now? I'm not seeing anything like this at pangeo-data.github.io/pangeo.

**Comment by mrocklin** [comment]
I'm downloading this branch and building locally

```
cd docs
make html
```

**Comment by tjcrone** [comment]
Okay thanks. Well, from the screenshot and the branding repo, I will chime in to say that I really like it! Especially the blue/green gradient. And the E that echos the Dask dashboard is very cool. A very minor point that probably nobody else would care about or even notice, is the several different radii for corners in that stylized E. I would consider looking at the Panton Black Caps E and making all the corners on each of the bars the same as the radius for all the other sharp corners in the font. (Kindof a font nerd here so no big deal if this point is ignored.)

**Comment by mrocklin** [comment]
While we're nitpicking (sorry @rabernat ) it also looks like the logo doesn't smoothly transition to a white border, which makes things a bit odd when used with a white background.  There is a sharp transition in the upper left.  Perhaps this is feedback to push upstream?

**Comment by rabernat** [comment]
My plan here is to fill in the basic template and write the important content. I'm not a designer, so tweaking these things is not a great use of my time. Once this update is merged, we can turn it over to @meganfitzsimons to polish the look.

**Comment by tjcrone** [comment]
I feel like there is a possibility that you were a designer in another life.

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+`The GitHub Revolution: Why We're All in Open Source Now <https://www.wired.com/2013/03/github/>`_
+by Mikeal Rogers.
+
+There are lots of great free and open resources to learn how to use scientific
+python.
+Here is a list of a few of our favorites.
+
+- list of
+- scientific python
+- online tutorials
```
Comment (rabernat) [inline docs/quickstart.rst]: What are people's favorite online resources for teaching scientific python?

File: docs/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+There are also some excellent books, including
+
+- list of
+- scientific python
+- books
```
Comment (rabernat) [inline docs/quickstart.rst]: And what about books? What do we want to recommend?

File: docs/quickstart.rst (hunk 1 above)
Comment (willirath) [reply docs/quickstart.rst]: Not limited to python but great as a concise set of rules to work by: The ten scientific coding commandments - - http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003285

And not strictly scientific python but great for getting people to understand concepts / ideas: The video tutorials by @CoreyMSchafer - - https://www.youtube.com/user/schafer5

**Comment by jhamman** [comment]
No objections from me. I look forward to seeing it in action and contributing to it myself.

**Comment by mrocklin** [comment]
No objections

**Pull request merged**